functions/activate-batch-background.js

```diff
@@ -32,6 +32,22 @@ async function handle(data, db, contract) {
     let result = await db.query('batches', 'batch', data.batch)
     const existingAddresses = result.Items
 
+    const allAccountsResponse = await db.scan('accounts', 5555);
```

Comment:
Cannot scan 5555, there are more than 5555 addresses due to passes being exchanged.

We can instead do batch query for every address in the batch.

Similar to the put below (but with get):
```
for(let i = 0;i < keyCount;i+=MAX_CONCURRENCY) {
      await Promise.all(holderKeys.slice(i, i+MAX_CONCURRENCY).map(async a => {
        const batchItem = prizeAssignment[a] ? {
            batch: data.batch,
            address: a,
            balance: Math.max(prizeAssignment[a].length, holderBalance[a]),
            prizes: JSON.stringify(prizeAssignment[a])
          } : {
            batch: data.batch,
            address: a,
            balance: holderBalance[a],
            prizes: '[]'
          }
        await db.put('batches', batchItem)

        const accountUpdateBadluckCount = {
          address: a,
          ...addressBadluckCount[a]
        }

        await db.put('accounts', accountUpdateBadluckCount)
      }))
```